develop/parachains/customize-parachain/add-existing-pallets.md

```diff
@@ -81,10 +81,9 @@ std = [
 ]
 ```
 
-This section specifies the default feature set for the runtime, which includes the `std` features for each pallet. When the runtime is compiled with the `std` feature set, the standard library features for all listed pallets are enabled. For more details on how the runtime is compiled as both a native binary (using `std`) and a Wasm binary (using `no_std`), refer to the [Wasm build](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/substrate/index.html#wasm-build){target=\_blank} section in the Polkadot SDK documentation.
+This section specifies the default feature set for the runtime, which includes the `std` features for each pallet. When the runtime is compiled with the `std` feature set, the standard library features for all listed pallets are enabled. If you forget to update the features section in the `Cargo.toml` file, you might encounter `cannot find function` errors when compiling the runtime.
```

Comment:
I don't know if this is correct. Do you have the source of this error?

Reply:
I literally took the text out of the note without changes beyond that so if it's wrong it was wrong before me. I am not familiar with Rust or this error myself.